Completion/Unix/Type/_tar_archive

```diff
@@ -20,8 +20,10 @@ if [[ "$1" = *[urtx]* ]]; then
     _files "$expl[@]" -g '*.(tar|TAR).bz2(-.)'
   elif [[ "$1" = *J* ]]; then
     _files "$expl[@]" -g '*.(tar|TAR).(lzma|xz)(-.)'
+  elif [[ "$1" = *zstd* ]]; then
+    _files "$expl[@]" -g '*.(tar|TAR).(zst|zstd)(-.)'
   elif [[ "$_cmd_variant[$service]" == (gnu|libarchive) ]]; then
-    _files "$expl[@]" -g '*.((tar|TAR)(.gz|.GZ|.Z|.bz2|.lzma|.xz|)|(tbz|tgz|txz))(-.)'
+    _files "$expl[@]" -g '*.((tar|TAR)(.gz|.GZ|.Z|.bz2|.lzma|.xz|.zst|.zstd)|(tbz|tgz|txz|tzst|tzstd))(-.)'
```

Comment:
Are `tzst` and and `tzstd` extensions really a thing or are you just making assumptions while copy/pasting? The original purpose behind `tgz`, `tbz` extensions was to cope with 8.3 filename limitations on old operating systems so they mostly are just a relic from the past. `tzst` doesn't even fit in 3 characters.

Reply:
Oh, today I learned. I had no idea it was more than just an abbreviation for the heck of it. So to answer your question: It was just an assumption that this kind of abbreviation should be continued. However, I see now that it doesn't really make sense. I'll remove it.